front/app/api/events/types.ts

```diff
@@ -18,6 +18,7 @@ export interface IEventData {
     created_at: string;
     updated_at: string;
     attendees_count: number;
+    online_link: string;
```

Comment:
Should we indicate that `online_link` is optional?
```suggestion
    online_link: string | undefined | null;
```